cheroot/test/test_core.py

```diff
@@ -316,25 +316,29 @@ def testSlashes(self):
         # Make sure GET params are preserved.
```

Comment:
Changes of this module are potentially useless, since in contains tests related to cherrypy, which are likely to be removed